maintenance/wikia/city_interwiki_links.sql

```diff
@@ -97,6 +97,7 @@ REPLACE INTO interwiki (iw_prefix,iw_url,iw_local,iw_trans) VALUES
 ('zeldapendium','http://www.zeldapendium.de/wiki/$1','0','0'),
 ('homepage','https://www.fandom.com/$1','1','0'),
 ('vstf','https://vstf.fandom.com/wiki/$1','1','0'),
```

Comment:
Should vstf be removed?

Reply:
Hmm, I think I would keep it for now so that we can use both vstf and soap prefixes. We could create next ticket that would deal with adding soap interwiki prefix for all existing wikis. What do you think?